src/common/actions/entity.actions.ts

```diff
@@ -61,6 +60,7 @@ const GENERAL_ACTIONS_GROUP: IActionGroup = {
       icon: '',
       class: 'mc-icon-Earth',
       titleTranslationId: 'action.viewer.tooltip',
+      dependentField: {field: 'isBeingDeleted', expectedValue: false},
```

Comment:
Let's consider to work with **productStatus** like(without additional model field) 
**_isBeingDeleted: (item as Layer3DRecordModelType).productStatus === RecordStatus.BEING_DELETED_**